x/subscription/spec/03_messages.md

```diff
@@ -0,0 +1,94 @@
+<!--
+order: 3
+-->
+
+# Messages
+
+## MsgCreatePlan
+
+Merchant or payment provider create plans.
+
+```protobuf
+message MsgCreatePlan {
+  string title;
+  string description;
+  Decimal price,  // price in fiat currency show to user
+  string fiat_currency,  // fiat currency name
+  string token_denom,  // denomination of token
+  Decimal conversion_rate,  // conversion rate from fiat currency to token, maintained by owner
+  int32 interval
```

Comment:
@thomasmktong-cryptocom maybe charging at "first day of the month" / "last day of the week" ...

Reply:
"first Monday of the month"

Reply:
I think there's some complexity to add calendar to support non-fixed intervals, I guess we can support fixed-seconds intervals first, support calendar intervals at next step.

Reply:
Specified both intervals(fixed seconds and calendar one), but at implementation side, we can support second interval first for simplicity.